src/index.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+const state = {
+  temperature: 79,
+  city: 'London',
+  isF: true,
+};
+
+const convertKtoC = (temp) => temp - 273.15;
+const convertKtoF = (temp) => (temp - 273.15) * (9 / 5) + 32;
+const convertCtoF = (temp) => temp * (9 / 5) + 32;
+const convertFtoC = (temp) => (temp - 32) * (5 / 9);
+
+const getLatAndLon = (city) => {
+  return axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: city,
+      },
+    })
+    .then((response) => {
+      console.log(response.data);
+      return {
+        lat: response.data[0].lat,
+        lon: response.data[0].lon,
+      };
+    })
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+    });
+};
+
+const getWeatherKelvin = (lat, lon) => {
+  return axios
+    .get('http://127.0.0.1:5000/weather', {
```

Comment:
You can use a constant variable like `WEATHER_URL` instead of string literal here. 

"Constants free the programmer from having to remember what each literal should be. Often values that stay constant throughout the program have a business meaning. If there are several such values, the programmer can define them all in the beginning of the program and then work with the easier-to-remember constant names." From: https://www.diffen.com/difference/Constant_vs_Literal